pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+<!--
+Include a summary of the change here.
+-->
+
+Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
We removed this section in MOTH as the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification should make it clear what type of change it is in the PR title.